app/controllers/commitments_controller.rb

```diff
@@ -51,6 +50,8 @@ def new
 
   def create
     @commitment = Commitment.new(commitment_params.merge(user: current_user, user_created: true))
+    @commitment.manager_ids = Criterium.find(@commitment.criterium_id).manager_ids
```

Comment:
the reason for keeping the has_and_belongs_to_many is that we aren't sure what will come in via the import. it's possible that the cbd endpoint is returning multiple manager types. we have a meeting next week to talk to them. if they will only return one manager type, then it should change. I basically wanted to avoid changing everything, then changing it back if possible.

The reason for copying the ids from the criterium to the commitment is that we need that data, but imported commitments do not have criteria records. alternative would be to build one with all the managers on, but that would be more difficult when importing via the csv. this way we can just keep the import the same

Reply:
Re: the copying, would something like `criterium ? criterium.manager_ids : @manager_ids` work (as `Commitment#manager_ids`)? So look at the criterium's manager IDs if there is an associated criterium, otherwise use the commitment's manager IDs which will be set for imported records? Not sure if that's overcomplicating it though